packages/table/src/headers/_headers.scss

```diff
@@ -254,3 +252,30 @@
   }
   // stylelint-enable selector-class-pattern
 }
+
+.bp-table-column-name,
+.bp-table-row-name {
+  transition: color $cell-transition-duration;
+}
+
+.bp-table-header.pt-loading {
+  @include vertical-center();
+
+  .bp-table-column-name,
+  .bp-table-row-name {
+    flex: 1;
+  }
+
+  .bp-table-column-name-text {
+    @include vertical-center();
+    padding: ($pt-grid-size * 1.1) $pt-grid-size ($pt-grid-size * 1.2);
```

Comment:
the designers were against 8px, too thick. there was a bushy eyebrow meme going on for a while, because that's what it looks like 😛.